add-on/panels/css/panel.css

```diff
@@ -96,6 +96,67 @@ body {
   border-radius: 3px;
 }
 
+/* Rename closing room panel */
+.rename-newRoom > img {
+  margin-left: 16px;
+  position: absolute;
+  width: 32px;
+}
+html[dir="rtl"] .rename-newRoom > img {
+  margin-right: 16px;
```

Comment:
You get away with this here because of the padding on the `.rename-container`, however you really want to set `margin-left: 0;` here.